README.md

```diff
@@ -167,9 +167,10 @@ Optional arguments are:
 
 * `-i` gives a name to a custom TNF container image. Supports local images, as well as images from external registries.
 * `-k` gives a path to one or more kubeconfig files to be used by the container to authenticate with the cluster. Paths must be separated by a colon.
-* `-n` gives the network mode of the container. Defaults to `bridge`. See the [docker run --network parameter reference](https://docs.docker.com/engine/reference/run/#network-settings) for more information on how to configure network settings.
+* `-n` gives the network mode of the container. Defaults set to `host`, if the node is in selinux enable then need to set the value to somthing that are not host. See the [docker run --network parameter reference](https://docs.docker.com/engine/reference/run/#network-settings) for more information on how to configure network settings.
```

Comment:
... 'host', which requires selinux to be disabled. Alternatively, 'bridge' mode can be used with selinux if TNF_CONTAINER_CLIENT is set to "docker" or running the test as root.